src/legacy.rs

```diff
@@ -193,4 +204,82 @@ mod aesgcm_tests {
         ).unwrap();
         assert_eq!(plaintext, "I am the walrus");
     }
+
+    // We have some existing test data in b64, and some in hex,
+    // and it's easy to make a second `try_decrypt` helper function
+    // than to re-encode all the data.
```

Comment:
These tests have just moved from `lib.rs` into `legacy.rs` to be next to their other `aesgcm` friends, I haven't changed them at all.